### DISCUSSION TITLE
ok idea from user2

### DISCUSSION THREAD

**Discussion by xbeumala2** [Marfeel/testXB#6] [Ideas]
ñlas fñal qñewlrk ñqw añlsk ñas ñkqw erñklq pñlk asñlf añl qqwer lñas fñlq qwerñ añl xi vzñl qweñrl ql asdfasdf2
 ñlas fñal qñewlrk ñqw añlsk ñas ñkqw erñklq pñlk asñlf añl qqwer lñas fñlq qwerñ añl xi vzñl qweñrl ql asdfasdf2
 ñlas fñal qñewlrk ñqw añlsk ñas ñkqw erñklq pñlk asñlf añl qqwer lñas fñlq qwerñ añl xi vzñl qweñrl ql asdfasdf2
 ñlas fñal qñewlrk ñqw añlsk ñas ñkqw erñklq pñlk asñlf añl qqwer lñas fñlq qwerñ añl xi vzñl qweñrl ql asdfasdf2

 ñlas fñal qñewlrk ñqw añlsk ñas ñkqw erñklq pñlk asñlf añl qqwer lñas fñlq qwerñ añl xi vzñl qweñrl ql asdfasdf2

 ñlas fñal qñewlrk ñqw añlsk ñas ñkqw erñklq pñlk asñlf añl qqwer lñas fñlq qwerñ añl xi vzñl qweñrl ql asdfasdf2
 ñlas fñal qñewlrk ñqw añlsk ñas ñkqw erñklq pñlk asñlf añl qqwer lñas fñlq qwerñ añl xi vzñl qweñrl ql asdfasdf2
 ñlas fñal qñewlrk ñqw añlsk ñas ñkqw erñklq pñlk asñlf añl qqwer lñas fñlq qwerñ añl xi vzñl qweñrl ql asdfasdf2 ñlas fñal qñewlrk ñqw añlsk ñas ñkqw erñklq pñlk asñlf añl qqwer lñas fñlq qwerñ añl xi vzñl qweñrl ql asdfasdf2

**Comment by xbeumala** [comment]
it should trigger a notification

**Comment by xbeumala** [comment]
here  we go... another answers
